packages/e2e-tests/specs/blocks/collumns.test.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	createNewPost,
+	getAllBlockInserterItemTitles,
+	insertBlock,
+	openAllBlockInserterCategories,
+	openGlobalBlockInserter,
+} from '@wordpress/e2e-test-utils';
+
+describe( 'Collumns', () => {
```

Comment:
Typo here:
```suggestion
describe( 'Columns', () => {
```

and in the name of the file.